docs/writing-rules/data-flow/data-flow-overview.md

```diff
@@ -2,15 +2,15 @@
 slug: data-flow-overview
 append_help_link: true
 description: >-
-  Semgrep can run data-flow analysis on your code, this is used for constant propagation and for taint tracking.
-sidebar_label: Engines overview
+  Semgrep can run data-flow analyses on your code, this is used for constant propagation and for taint tracking.
+sidebar_label: Engine overview
 ---
 
 import DataFlowStatus from "/src/components/concept/_data-flow-status.mdx"
 
-# Data-flow analysis engines overview
+# Data-flow analyses engine overview
 
-Semgrep provides intra-procedural data-flow analysis engines that open various Semgrep capabilities. There are two main data-flow analysis engines:
+Semgrep provides an intra-procedural data-flow analyses engine that opens various Semgrep capabilities. Semgrep provides the following data-flow analyses:
```

Comment:
```suggestion
Semgrep provides an intra-procedural data-flow analysis engine that opens various Semgrep capabilities. Semgrep provides the following data-flow analyses:
```

Reply:
@IagoAbal since you suggested this edit, I rather changed the plural of analyses in the Overview document also (to be consistent). "Data-flow analyses" may sound confusing, users do not actually need to know that one engine makes many analyses... unless necessary for the use of the product.